sql/updates/0.10.1--0.11.0-dev.sql

```diff
@@ -0,0 +1 @@
+DROP FUNCTION IF EXISTS _timescaledb_internal.dimension_get_time(integer);
```

Comment:
depending on which release this function ends up in, we should remember to put this in relevant update file.